lib/dpl/provider/atlas.rb

```diff
@@ -16,14 +16,16 @@ class Atlas < Provider
             chmod +x $HOME/bin/gimme
           fi
 
-          export GOPATH="$HOME/gopath:$GOPATH"
-          eval "$(gimme 1.4.2)" &>/dev/null
+          if [ -z $GOPATH ]; then
+            export GOPATH="$HOME/gopath"
+          else
+            export GOPATH="$HOME/gopath:$GOPATH"
+          fi
+          eval "$(gimme 1.6)" &> /dev/null
 
+          unset GIT_HTTP_USER_AGENT
```

Comment:
Is it safe to unset it here? I _think_ it is, but do keep in mind that another provider might depend on this value later. (I don't think that this `unset` does affects the `dpl` calls later.)